src/UI/jQuery/jquery.ajax.js

```diff
@@ -14,7 +14,7 @@ module.exports = function() {
             xhr.url += $.param(xhr.data);
             delete xhr.data;
         }
-        if (xhr) {
+        if (xhr && !(xhr.headers instanceof Object && Object.isFrozen(xhr.headers))) {
```

Comment:
It's likely better to create an alternative ajax function, such as $.ajaxOriginal(), for these cases.